igmspec/ingest/myers.py

```diff
@@ -4,13 +4,143 @@
 
 import numpy as np
 import os
-import pdb
-
+#import pdb
+from xastropy.xutils import xdebug as xdb
 from astropy.table import Table
 from astropy.io import fits
+from astropy import units as u
+
+# What should this matching tolerance be?? Set to 2.0" for now
+def add_to_hdf(hdf, Z_MIN = 0.1, Z_MAX = 7.1, MATCH_TOL = 2.0*u.arcsec):
+
+    from astropy import units as u
+    from astropy.table import QTable, Table, Column, hstack, vstack
+    from astropy.coordinates import SkyCoord, match_coordinates_sky, search_around_sky
+
+    ## SDSS/BOSS data stuff
+    specfile = os.getenv('RAW_IGMSPEC') + '/sdss/specObj-dr12_trim.fits'
```

Comment:
My sdss folder is named SDSS.  But I think we actually want this
in the BOSS folder.  

Rename.